devspace.yaml

```diff
@@ -68,19 +68,102 @@ commands:
       done
 
 deployments:
+  - name: argocd
+    kubectl:
+      kustomize: true
+      manifests:
+        - ./manifests/edgefarm.core/argocd
+    namespace: argocd
+
+  - name: keycloak
+    kubectl:
+      manifests:
+        - ./manifests/edgefarm.core/keycloak
+    namespace: my-keycloak-operator
+
   - name: kubeedge
     namespace: kubeedge
     helm:
       chart:
         name: charts/kubeedge
       values:
+        cloudcore:
+          image: kubeedge/cloudcore:v1.8.2
         cloudHub:
           dnsNames:
             - ${HOST_IP_NIP_IO}
+          websocket:
+            port: 10000
+          https:
+            port: 10002
+          stream:
+            port: 10003
+          tunnel:
+            port: 10004
+          annotations: {}
```

Comment:
why do you configure the values here? as this is already done in `charts/kubeedge/values.yaml`?